substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -1964,7 +1971,8 @@ private List<DebugLocalValueInfo> initSyntheticInfoList(ParamLocationProducer lo
                 JavaKind kind = ownerType.getJavaKind();
                 JavaKind storageKind = isForeignWordType(ownerType, ownerType) ? JavaKind.Long : kind;
                 assert kind == JavaKind.Object : "must be an object";
-                NativeImageDebugLocalValue value = locProducer.nextLocation(kind);
+                // NativeImageDebugLocalValue value = locProducer.nextLocation(kind);
```

Comment:
Do we still need this comment here and a few lines below?

Reply:
No. Removed.